shared/AppInsightsCommon/src/Util.ts

```diff
@@ -213,7 +213,7 @@ export class Util {
     public static getSessionStorageKeys(): string[] {
         const keys = [];
 
-        if (Util.canUseSessionStorage()) {
+        if (Util.canUseSessionStorage() && typeof window === "object") {
```

Comment:
Same here add typeof check inside canUseSessionStorage method

Reply:
double checked, window is checked in _getVerifiedStorageObject method that is called in canUseSessionStorage method. So no need to check it again.